app/models/work.rb

```diff
@@ -186,8 +186,19 @@ def check_for_invalid_chapters
   after_update :adjust_series_restriction
   
   after_save :expire_caches
-  
+
+  def self.rss_work_summary_key(id)
+    "/v1/rss_data_tags/#{id}/summary"
+  end
```

Comment:
Are these deliberately in the same space as the tag keys despite using a different id field? What I mean is, you can theoretically have a work and tag with the same id, which would mean you had rss_data_tags/[work id]/whatever and rss_data_tags/[tag id] and if you wanted to wipe out everything under rss_data_tags/[work id] you would as a result take out everything for the tag that accidentally shares the id number. This seems clearly wrong so I'm wondering what I'm missing?